example/example-dev.html

```diff
@@ -205,10 +212,15 @@
             }
           },
           {
-            type : 'paragraph',
-            id: "b6ji-DvaKb",
-            data : {
-              text : 'Hey. Meet the new Editor. On this page you can see it in action — try to edit this text. Source code of the page contains the example of connection and configuration.'
+            "id": "b6ji-DvaKb",
+            "type": "paragraph",
+            "data": {
+              "text": "Hey. Meet the new Editor. On this page you can see it in action<sup>1</sup> — try to edit this text. Source code of the page contains the example of connection and configuration."
+            },
+            "tunes": {
```

Comment:
I would suggest not complicating the existed example. Let's move it to the separate example page, `example-tunes`, for example